Update unit test for drive private variables

Maaz0070 opened the pull request.

Maaz0070 committed:
fixed error in base class rover_system

Maaz0070 committed:
merged changes with master

Maaz0070 committed:
changes to RoverSystem base class

Maaz0070 committed:
changes to RoverSystem base class

Maaz0070 committed:
merging with unit testing commit

Maaz0070 committed:
removed common mc data to base class

Maaz0070 committed:
removed common mc data to base class

Maaz0070 committed:
merged with master

Maaz0070 committed:
Switchable leg orientation implmeented

Maaz0070 committed:
moved library files into common folder

Maaz0070 committed:
made sure wheels are not moving before swithing orientation

Maaz0070 committed:
removed testing line in main file

Maaz0070 committed:
removed util folder from Common

Maaz0070 committed:
edited leg orientation using arrays

Maaz0070 committed:
merged with master

Maaz0070 committed:
removed testing line in drive system

Maaz0070 committed:
used IsStopped in SwitchLegOrientation method

Maaz0070 committed:
Merge branch 'master' of https://github.com/SJSURoboticsTeam/urc-control_systems-2021 into Maaz-LegOrientation

Maaz0070 committed:
fixed test case for leg orientation

Maaz0070 committed:
Merge branch 'master' of https://github.com/SJSURoboticsTeam/urc-control_systems-2021 into Maaz-LegOrientation

Maaz0070 committed:
edited testing function

FluffyFTW committed:
changed public member variables to private

FluffyFTW committed:
Revert "changed public member variables to private"

This reverts commit 5d33258beaad9ea698d00888fc6f28c1d3360d7a.

FluffyFTW committed:
changed all wheels to fit the struct and added a getter for the wheels

FluffyFTW committed:
Revert "changed all wheels to fit the struct and added a getter for the wheels"

This reverts commit c47566ec97c94549045544ce7ab0ee04deaf94d3.

Commit:
fixes broken unit test

Commit:
handles merge conflicts

Commit:
fixes variable naming error

Maaz0070 committed:
updated unit test to account for private wheels struct in Drive System Class

Maaz0070 committed:
merged with branch

Maaz0070 committed:
fixed merge conflicts in Drive System file

naterpotatoers reviewed (changes requested):
I think these changes are already merged into master?

Maaz0070 committed:
merged with master

Maaz0070 commented:
>
The constructor for the Drive System file was changed; we added a struct for the wheels; and updated unit tests to work with the struct passing. There are changes to this PR that weren't in the other one.

naterpotatoers reviewed (approved):
okay looks good then

Maaz0070 committed:
updated unit test with struct

Maaz0070 committed:
Merge branch 'master' of https://github.com/SJSURoboticsTeam/urc-control_systems-2021 into Update-Unit-Test-for-Drive-Private-variables

Maaz0070 committed:
merged with master
